README.md

```diff
@@ -2,11 +2,10 @@
 
 
 [![](https://img.shields.io/badge/made%20by-Protocol%20Labs-blue.svg?style=flat-square)](http://ipn.io)
-[![](https://img.shields.io/badge/project-IPFS-blue.svg?style=flat-square)](http://ipfs.io/)
+[![](https://img.shields.io/badge/project-BTFS-blue.svg?style=flat-square)](https://www.bittorrent.com/btfs/)
 [![standard-readme compliant](https://img.shields.io/badge/standard--readme-OK-green.svg?style=flat-square)](https://github.com/RichardLitt/standard-readme)
```

Comment:
could you remove all the badges? most don't apply to us.

Reply:
ok